README.Rmd

```diff
@@ -40,8 +40,104 @@ You can install the development version of elicitr from GitHub with:
 pak::pak("CREWdecisions/elicitr")
 ```
 
+### Getting started
+
+```{r}
+library(elicitr)
+```
+
+All the functions in the elicitr package start with the prefix `elic_`. After that, two prefixes are available: `elic_cont` and `elic_cat`. This design choice is intended to enhance functions discovery.\
+`elic_cont` functions are used for the elicitation of continuous variables and `elic_cat` functions for the elicitation of categorical variables.
+
+### Elicitation of continuous variables
+
+#### Simulated datasets
+
+Two simulated datasets are included in elicitr. These datasets are intended to demonstrate the functionality of the package and do not represent an actual elicitation process (names are also randomly generated).
+
+```{r}
+round_1
+```
+###
+```{r}
+round_2
+```
+
+#### How elicitr works
+
+For the analysis of continuous variables, the core of the elicitr package is an object of class `elic_cont`. Just as you create a form to collect estimates in an elicitation process, you need to create the `elic_cont` object to store metadata information. This allows to check whether all experts have given their answers in the expected way.\
```

Comment:
I separated the overview information (put it before the use cases. What do you think?

Reply:
It looks great now! I would only use a period instead of a colon at the end of the sentence. Another minor change is to wrap the function names with back ticks. Although they are not real function names but only pseudo names, they are still functions and the back tick make it more clear.

Reply:
You forgot the plot function.

Reply:
I added it.